backend/ionic.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import requests
+import uuid
+
+
+IONIC_URL = 'https://api.ionic.io'
+
+
+def BuildUrl(suffix):
+  return '%s/%s' % (IONIC_URL, suffix)
+
+
+class Ionic(object):
+  """Ionic API Wrapper."""
+
+  def __init__(self, app_id, security_tag, access_token):
+    self.app_id = app_id
+    self.access_token = 'Bearer %s' % access_token
+    self.security_tag = security_tag
+    self.Test()
+
+  def GetAuthHeaders(self):
+    """Generate authoriztaion headers."""
+    return {
+      'Authorization': self.access_token,
+      'Content-Type': 'application/json',
```

Comment:
I'm pretty sure the requests module puts this header in for you when you send JSON data.
Also, does requests require updating the requirements.txt?